service/src/test/java/com/ge/predix/controller/test/AttributeConnectorControllerIT.java

```diff
@@ -86,6 +86,15 @@ public void setup() {
     }
 
     @Test(dataProvider = "requestUrlProvider")
+    public void attributeConnectorZoneDoesNotExistException(final String endpointUrl) throws Exception {
```

Comment:
No need to test this with data provider.  One pass is enough.  Also, there is testZoneDoesNotExist() below.  How is it different?